compact/tracking/mpgd_barrel.xml

```diff
@@ -21,70 +21,91 @@
     </comment>
 
     <constant name="InnerMPGDBarrelLayer_length"            value="InnerMPGDBarrel_length" />
-    <constant name="InnerMPGDBarrelLayer_thickness"         value="5*cm" />
-    <constant name="InnerMPGDBarrelLayer_rmin"              value="InnerMPGDBarrel_rmin"/>
-    <constant name="InnerMPGDBarrelLayer_rmax"              value="InnerMPGDBarrelLayer_rmin + InnerMPGDBarrelLayer_thickness"/>
+    <constant name="InnerMPGDBarrel_hack"                   value="0*cm"/>
```

Comment:
I haven't followed the whole history of the file: this "hack" was there when I first got involved and I have understood it since as some kind of adjustment variable. But giving it now a second thought, I agree w/ you: it can be removed.

Reply:
I agree this can be removed. I think it is a left over debug artifact from when Niv was working on the geometry

Reply:
Done. See commit #7403586 ("https://github.com/eic/epic/commit/7403586").